RecoJets/JetProducers/plugins/SubEventGenJetProducer.cc

```diff
@@ -38,7 +38,7 @@ namespace {
 SubEventGenJetProducer::SubEventGenJetProducer(edm::ParameterSet const& conf) : VirtualJetProducer(conf) {
   //   mapSrc_ = conf.getParameter<edm::InputTag>( "srcMap");
   ignoreHydro_ = conf.getUntrackedParameter<bool>("ignoreHydro", true);
-  produces<reco::BasicJetCollection>();
+
```

Comment:
What is this going to `produces` now?

Reply:
The code is producing  a collection of type
 vector<reco::GenJet>
If I add the line:
produces<reco::GenJetCollection>();
I get an exception complaining about a 
"Duplicate Product Identifier"